torch/csrc/jit/serialization/unpickler.cpp

```diff
@@ -57,6 +57,7 @@ void restoreAccurateTypeTags(const IValue& root, const TypePtr& type_tag) {
       case StorageType::Kind:
       case NumberType::Kind:
       case FloatType::Kind:
+      case ComplexDoubleType::Kind:
```

Comment:
I guess this already existed but this naming looks inconsistent -- aren't these referring to python types, so this should be ComplexType?

Reply:
I was under the impression that there will eventually be a `ComplexFloatType` as well but if not, then I agree that it should be `ComplexType`.

Reply:
hmm I didn't know that we follow the python naming here. yeah should be `ComplexType`!